### ISSUE TITLE
Inverted SBUS

### ISSUE THREAD

**Issue by rinozen** [Sokrates80/sbus_driver_micropython#1]
Hi.. Thanks for the code..
A question, do i need to get uninverted SBUS signal before connect it to Pyboard RX (UART)?

**Comment by Sokrates80** [comment]
Yes, there is an example of diy shield on the project website: air-pay.com

Cheers

**Comment by Sokrates80** [comment]
I meant air-py.com

**Comment by rinozen** [comment]
@Sokrates80 , is the library work on esp8266?

**Comment by matzetronics** [comment]
Thanks for the code!
Just a quick note that the software runs perfectly on the RPi Pico, which has the ability to invert the RX Pin in Code:
```
    def __init__(self, uart_port):
        self.sbus = UART(0,baudrate=100000, parity=0, invert=UART.INV_RX, timeout_char = 3, stop=2, tx=Pin(16), rx=Pin(17))

```
Nothing else needs to be changed and the signal can be directly connected to the Pico (GPIO 17 in this case)

Btw, works well with the SBUS receiver from the DJI Phantom 2 Vision+
